infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LayoutSwitchDao.kt

```diff
@@ -477,10 +478,12 @@ class LayoutSwitchDao(
     }
 
     fun findLocationTracksLinkedToSwitchAtMoment(
+        layoutBranch: LayoutBranch,
         switchId: IntId<TrackLayoutSwitch>,
         topologyJointNumber: JointNumber,
         moment: Instant,
     ): List<LocationTrackIdentifiers> {
+        assertMainBranch(layoutBranch)
```

Comment:
Kurkkasin tuota vielä ja näyttäisi että tätäkin käytetään vain ratkosta, ja se on muutenkin vielä TBD-osastoa joten ok näin.